palace/fem/coefficient.hpp

```diff
@@ -139,16 +144,17 @@ class BdrSurfaceFluxCoefficient : public mfem::Coefficient,
   const MaterialOperator &mat_op;
   bool two_sided;
   const mfem::Vector &x0;
+  const double scaling;
```

Comment:
Could we maybe add `scaling` to the base class `BdrGridFunctionCoefficient` instead of each individual subclass?